src/ARte/users/static/css/upload.css

```diff
@@ -46,4 +46,26 @@
 
 #positionTitle:hover + #tip2 {
     display: inline-block;
-}
\ No newline at end of file
+}
+
+#content-box {
```

Comment:
Awesome work on this PR, but this values on the content-box are weird in my screen, maybe using percents instead of vh would ensure that it works on every screen, can you update that?
```suggestion
#content-box {
    height: 100%;
    width: 100%;
}
```

This was my result with the values in vh.
![image](https://user-images.githubusercontent.com/8556291/95517140-cfbdd100-0996-11eb-88d1-48ef0f4aba06.png)